data/styles/categories.css

```diff
@@ -27,6 +27,9 @@
         0 3px 5px alpha(black, 0.1);
     font-size: 32px;
     font-weight: 300;
+    margin: 1.333rem; /* 12px */
+    padding: 1.333rem;
+    min-height: 3em;
```

Comment:
You changed ems to px elsewhere but used ems here - is that OK?

Reply:
Yeah it works for margins and padding, but when used as stops in gradients we get weird rounding sometimes that makes gradients either too hard or lines disappear etc